ComputerInterface/BaseGameInterface.cs

```diff
@@ -257,6 +290,11 @@ public static void InitVoiceMode()
             SetVoiceMode(GetVoiceMode());
         }
 
+        public static void InitItemMode()
+        {
+            SetVoiceMode(GetVoiceMode());
```

Comment:
Are these the wrong function calls?
```suggestion
            SetItemMode(GetItemMode());
```